Tribler/Main/tribler_main.py

```diff
@@ -943,7 +943,6 @@ def OnExit(self):
             self._logger.info("ONEXIT Session is shutdown")
 
         self.utility.session.notifier.notify(NTFY_CLOSE_TICK, NTFY_INSERT, None, 'Deleting instances')
-        wx.Yield()
```

Comment:
If I remember correctly, removing this `yield` causes the shutdown splash screen to not correctly update. I think it can be removed now however since we will not release the wx3 version.